eZ/Publish/API/Repository/Events/URLWildcard/UpdateEvent.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\API\Repository\Events\URLWildcard;
+
+use eZ\Publish\API\Repository\Values\Content\URLWildcard;
+use eZ\Publish\API\Repository\Values\Content\URLWildcardUpdateStruct;
+use eZ\Publish\SPI\Repository\Event\AfterEvent;
+
+final class UpdateEvent extends AfterEvent
+{
+    /** @var \eZ\Publish\API\Repository\Values\Content\URLWildcard */
+    private $URLWildcard;
+
+    /** @var \eZ\Publish\API\Repository\Values\Content\URLWildcardUpdateStruct */
+    private $updateStruct;
+
+    public function __construct(
+        URLWildcard $urlWildcard,
+        URLWildcardUpdateStruct $updateStruct
+    ) {
+        $this->URLWildcard = $urlWildcard;
```

Comment:
Related to @adamwojs comment, please unify the naming (make it `$urlWildcard*`) as here it is obvious inconsistency. :)

Reply:
Changed